### PR TITLE
[Coverimage, plugin] Colons in menus; unlock sizespinner 

### PR DESCRIPTION
**Pull request by zwim**
This PR changes the menu entries to colons and unlocks cache sizespinners, when cover image is not activated.

Honor recommendation of using a default value as in https://github.com/koreader/koreader/pull/7371#issuecomment-788157906

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/8430)

### CODE REVIEW

File: plugins/coverimage.koplugin/main.lua [hunk 1]
```diff
@@ -72,15 +72,17 @@ local default_fallback_path = DataStorage:getDataDir() .. "/"
...
-    self.cover_image_cache_maxsize = G_reader_settings:readSetting("cover_image_cache_maxsize") or 5 -- MB
+    self.cover_image_format = G_reader_settings:readSetting("cover_image_format", "auto")
+    self.cover_image_quality = G_reader_settings:readSetting("cover_image_quality", 75)
+    self.cover_image_grayscale = G_reader_settings:isTrue("cover_image_grayscale")
+    self.cover_image_stretch_limit = G_reader_settings:readSetting("cover_image_stretch_limit", 8)
+    self.cover_image_background = G_reader_settings:readSetting("cover_image_background", "black")
+    self.cover_image_fallback_path = G_reader_settings:readSetting("cover_image_fallback_path",
+        default_fallback_path)
+    self.cover_image_cache_path = G_reader_settings:readSetting("cover_image_cache_path",
+            default_cache_path)
```
Comment (Frenzie) [inline plugins/coverimage.koplugin/main.lua:83]: nit

**Review by Frenzie — APPROVED** [review]

**Comment by zwim** [comment]
@Frenzie : May I merge this and #8431 now, or should I wait another day?

**Comment by Frenzie** [comment]
Approved means good to go in my eyes (however, I might've left the nitpick as part of an approval, indicating that I think it's good besides the one nit) and it's fairly small, so shouldn't need more eyes. :+1:

**Comment by Frenzie** [comment]
PS A comment as part of an approval might also not be a nit but simply an up to your discretion what you do with the comment.

**Pull request merged**